### ISSUE TITLE
Regression in 5.1.403 version for IfcWallStanardCase.

### ISSUE THREAD

**Issue by CCT-Mukund-Thakare** [xBimTeam/XbimGeometry#430]
*Opening in IfcWallStandardCase is not visible*

#### Assemblies  and versions affected:
Xbim.geometry Assembly 5.1.0.0 is affected.
Xbim.geometry Version 5.1.403 is affected. 

#### Steps (or code) to reproduce the issue:
Open the file in the xbim desktop application ( Assembly version 4.0.0.0, file version 4.0.19 ) - 
We can see as added in the below screen shot the opening in the IfcWallStandardCase is considered and we can see through the door.
![image](https://github.com/xBimTeam/XbimGeometry/assets/115206745/6a709b38-1a1b-4d3c-a082-bca2b8d42d49)

When we build ad run the XbIm.WindowsUI with latest code available in git hub we can see the opening is not considered.
As shown in below screen shot, there is no opening in wall, we can't see through the door.

![image](https://github.com/xBimTeam/XbimGeometry/assets/115206745/6cc0b809-74dd-4927-9dee-bfdb9b19fcbb)
 
#### Minimal file to reproduce the issue:
Attached the file to reproduce the bug.
[Solibriビル.zip](https://github.com/xBimTeam/XbimGeometry/files/11767342/Solibri.zip)

#### Expected behavior:
In the both version of assembly we should see the opening in wall, and can see through the door.

#### Actual behavior or exception details:
We can see in older version we can see through the door, and in latest version we can not see through the door becuase the opening is not considered.

#### Additional Details
I believe it is regression in the latest version.

I have reported similar issue adding the link - 
[ssues-418](https://github.com/xBimTeam/XbimGeometry/issues/418
)

Thank you !!

**Comment by andyward** [comment]
Note this is fixed in v6. But i did reproduce in latest 5.1.742 GE.

<img width="583" alt="image" src="https://github.com/xBimTeam/XbimGeometry/assets/923699/7a1bd59c-e876-4d28-b0fb-548092e4642d">

**Comment by mukundthakare** [comment]
Hello @andyward 

**How can I use V6 in my current project ?**
      Environemnt specifications of my project - 
            .net core 8.0.
             visual studio 2022
             
**What did I do to use V6 in my project ?**
After reading this particular post - https://github.com/xBimTeam/XbimGeometry/issues/461
I have shifted to the feature\netcore branch of XbimGeometry repo, is this the correct branch to use V6 ?

![image](https://github.com/xBimTeam/XbimGeometry/assets/38717053/774be46e-2a72-4e8e-ab22-de4962dd0c5e)

I am able to build solution on this branch and able to generate the required Xbim.GeometryEngine.dll.
But when I use this dll's in the project ,I get the error that, XBimGeometryEngine64.dll can not be loaded.

The version of packges that I use in the projects are.
![image](https://github.com/xBimTeam/XbimGeometry/assets/38717053/0735896a-8048-4626-b975-f9a3b7472526)

The relase folder
release\net8.0-windows looks like this.
![image](https://github.com/xBimTeam/XbimGeometry/assets/38717053/627b7b63-6baf-4c1c-931a-e40556481eb3)

If you notice, On this versions I have 2 dll's XbimGeometryEngine32.dll and XbimGeometryEngine64.dll which I am using.
But when I create dll's on **netcore branch** it is named with XbimGeometryEngine.dll
So I renamed it to XbimGeometryEngine32.dll and XbimGeometryEngine64.dll respectively to use in the project.
But it seem's it is not able to load those dll's.

So
Yeah please let me know if this is the correct branch to use v6 and how to use build dll's in the project.

Thank you !!

**Comment by andyward** [comment]
Yes that's the correct branch for v6 Geometry, but it looks like your application is mixing up both 5.1 and v6 xbim assemblies - and you seem to still have v5.1 Geometry in your dependencies.

 XbimGeometryEngine32.dll and XbimGeometryEngine64.dll are 5.1 geometry engine DLLs. In v6 we have different convention: the native Geometry dlls are deployed to win10-x64 and win10-x86 folders alongside the Xbim.Geometry.Engine.Interop.dll

You should upgrade Essentials to the latest 6.0.* version on Myget: https://www.myget.org/feed/xbim-develop/package/nuget/Xbim.Essentials

For Geometry you'll have to package locally and host from a local/private nuget. (I'm assuming you're building from source). If you want a pre-built v6 GE we have a private repo I can add you to.

**Comment by CCT-Mukund-Thakare** [comment]
Hello @andyward 
Thank you for you assistance
I am able to successfully upgrade my project to version 6 and build geometry package locally.

I can also see the mentioned issue fixed on the version 6.

There is one more issue, I had reported previously which is same as this one.
Link - https://github.com/xBimTeam/XbimGeometry/issues/418

I was hoping that with current issue-430 the previous issue-418 will also get fixed on V6 because they are same, but unfortunately issue-418 is not fixed with V6.

Let me know your thoughts on this.
Thank you !!
